include/common_structs.h

```diff
@@ -92,6 +92,12 @@ typedef struct CamPosSettings {
     /* 0x10 */ Vec3f position;
 } CamPosSettings; // size = 0x1C
 
+typedef struct DmaTable {
+    /* 0x00 */ s32 dmaStart;
+    /* 0x04 */ s32 dmaEnd;
+    /* 0x08 */ s32 dmaDest;
+} DmaTable;
```

Comment:
I think there's a nusys struct that looks like this

Reply:
If it's the one I'm thinking of, it's much larger than this